media-gfx/frogr/frogr-1.7.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils meson xdg
+
+DESCRIPTION="flickr applications for GNOME"
+HOMEPAGE="https://live.gnome.org/Frogr"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	>=dev-libs/glib-2.44:2
+	>=dev-libs/json-glib-1.2
+	>=dev-libs/libgcrypt-1.5:*
+	>=dev-libs/libxml2-2.6.8:2
+	media-libs/gstreamer:1.0
+	>=media-libs/libexif-0.6.14
+	>=net-libs/libsoup-2.34:2.4
+	>=x11-libs/gtk+-3.16:3[introspection]
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	dev-util/itstool
+	>=sys-devel/gettext-0.19.7
+	virtual/pkgconfig
+"
+# TODO add a useflag for enable-video or header-bar???
+
+pkg_postinst() {
+	xdg_pkg_postinst
+	gnome2_icon_cache_update
```

Comment:
This doesn't exist in the *gnome2-utils* eclass anymore, it's in *xdg-utils* and can be called with *xdg.eclass* that you inherit.